tests/ConditionSqlTest.php

```diff
@@ -286,6 +286,63 @@ public function testExpressionJoin(): void
         static::assertSame('+123 smiths', $mm2->get('contact_phone'));
     }
 
+    public function testExpressionJoinForeignCustomIdField(): void
```

Comment:
does this test adds anything to the coverage or how it is helpfull?

Reply:
Nope, removed